paper.tex

```diff
@@ -399,7 +399,7 @@ \section*{Introduction}
 Decades of pedagogical research has yielded many insights into
 how best to build and deliver lessons \cite{hlw}.
 Unfortunately,
-since most college and university faculty have little or no training in education,
+since most college and university faculty have little or no training in education \cite{browning2012},
```

Comment:
The use of "most" implies > 50%. I can't find in Browning2012 anything that supports the use of the word "most". The authors even say:
> In terms of training, **many** faculty have indicated they feel ill-equipped to change the way they teach and thus would like access to structured, formal training.
(emphasis mine)